torch/hub.py

```diff
@@ -64,15 +64,13 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 # Copied from tools/shared/module_loader to be included in torch package
```

Comment:
This comment is no longer true, right?

Reply:
Hum...I'm not sure.

Reply:
Good to leave the comment; this isn't a very substantial change, the comment may still be useful.